### PR TITLE
Add Bitcoin blockchain analysis tutorial

### PR DESCRIPTION
**Pull request by zseta**
# Description

Add a Bitcoin blockchain analysis tutorial to the docs

### CODE REVIEW

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+FROM one_hour_transactions
+WHERE bucket > NOW() - INTERVAL '1 day'
+ORDER BY 1
+```
+![volume and BTC-USD](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/volume_btc_usd.png)
+
+Again, the green line shows the average transaction volume over time and the
+yellow line shows the BTC-USD conversion rate at any given time.
+
+Next, let's segue into block-level insights by analyzing the connection
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:243]:
```suggestion
Next, get block-level insights by analyzing the connection
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+between transactions and blocks.
+
+### Do more transactions in a block mean the block is more expensive to mine?
+For someone who is not an expert on how the Bitcoin blockchain works, it might be a
+difficult question to answer. So let’s ask the data and see if it’s true.
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:248]: Why is this necessary? Is this something that is often claimed, but that is not intuitive to see? If so, I would just say that. Something like:

`It's often said that a block with more transactions is more expensive to mine. Analyze the data to see if it supports this trend.`

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+For this kind of analysis you might want to look at a larger time frame,
+so let’s change the analyzed time range to the last five days.
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:250]:
```suggestion
For this kind of analysis, you might want to look at a larger time frame.
Change the analyzed time range to the last five days.
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+SELECT
+ bucket as "time",
+ avg(tx_count) AS transactions,
+ avg(block_fee_sat)*0.00000001 AS "mining fee"
+FROM one_hour_blocks
+WHERE bucket > now() - INTERVAL '5 day'
+GROUP BY bucket
+ORDER BY 1
+```
+![transactions in block and mining fee](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/tx_in_block_expensive.png)
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:262]:
```suggestion
![Line graph with two lines showing the average number of transactions in a block and the block mining fee, over the last five days](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/tx_in_block_expensive.png)
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+
+Unsurprisingly, there’s a high correlation between the number of transactions
+in a block and the mining fee. The more transactions a block has, the higher
+the block mining fee.
+
+Let’s dive deeper and see if there is the same correlation between block
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:268]:
```suggestion
In the next query, see if there is the same correlation between block
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 6]
```diff
@@ -0,0 +1,393 @@
...
+SELECT
+ bucket as "time",
+ avg(block_weight) as "block weight",
+ avg(block_fee_sat*0.00000001) as "mining fee"
+FROM one_hour_blocks
+WHERE bucket > now() - INTERVAL '5 day'
+group by bucket
+ORDER BY 1
+```
+![block weight and mining fee](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/weight_fee.png)
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:282]:
```suggestion
![Line graph with two lines showing the block weight and the block mining fee, over the last five days](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/weight_fee.png)
```
Same comment as for last graph

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 7]
```diff
@@ -0,0 +1,393 @@
...
+
+And yes—you can see the same kind of high correlation between block weight
+(defined in weight units) and mining fee. Except when the block weight gets
+close to its maximum value (4M weight units) in which case it's impossible for a
+block to include more transactions.
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:287]:
```suggestion
You can see the same kind of high correlation between block weight
(defined in weight units) and mining fee. The relationship weakens when the block weight gets
close to its maximum value (4 million weight units), in which case it's impossible for a
block to include more transactions.
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 8]
```diff
@@ -0,0 +1,393 @@
...
+
+In the previous charts, you saw how mining fees are correlated to block
+weights, transaction volumes, etc. But now, let’s analyze the data from a
+different perspective. Miner revenue is not only made up of miner fees, it
+also includes block rewards after mining a new block. This reward is currently
+6.25 BTC and it gets halved every four years. Let’s analyze miner revenue!
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:293]:
```suggestion
In the previous charts, you saw how mining fees are correlated to block
weights and transaction volumes. In the next query, analyze the data from a
different perspective. Miner revenue is not only made up of miner fees. It
also includes block rewards after mining a new block. This reward is currently
6.25 BTC, and it gets halved every four years. What are some trends in miner revenue?
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 9]
```diff
@@ -0,0 +1,393 @@
...
+
+### What percentage of the average miner's revenue comes from fees vs. block rewards?
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:295]:
```suggestion
### What percentage of the average miner's revenue comes from fees compared to block rewards?
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 10]
```diff
@@ -0,0 +1,393 @@
...
+Miners are incentivized to keep the network up and running because they earn
+fees and rewards after mining each block. But let’s look at how much of their
+revenue comes from these two sources.
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:298]:
```suggestion
fees and rewards after mining each block. How much of their
revenue comes from each source?
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 11]
```diff
@@ -0,0 +1,393 @@
...
+```
+![fees and block reward](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/revenue_ratio.png)
+(THIS CHART NEEDS AN UPDATE)
+
+This chart analyzes the last 5 days of average miner revenue. The left
+axis is a percentage value that indicates the portion of total revenue that
+comes from transaction fees (green) and block rewards (yellow). It might be
+surprising for some, but most miner revenue comes from block rewards
+(6.25BTC at the moment). The fees portion never went above 3 % in the
+last 5 days.
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:322]:
```suggestion
This chart analyzes the last five days of average miner revenue. The left
axis shows the percentage of total revenue that
comes from transaction fees (green) and block rewards (yellow). Most miner
revenue actually comes from block rewards
(6.25&nbsp;BTC at the moment). Fees never accounted for more than 3% in the
last five days. 
```

Five or 5, doesn't really matter, just make it consistent. I've seen five everywhere else so went with that

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 12]
```diff
@@ -0,0 +1,393 @@
...
+SELECT
+   bucket as "time",
+   avg(block_fee_sat)*0.00000001 AS "fees",
+   FIRST((c.coinbase_tx - block_fee_sat), bucket)*0.00000001 AS "reward"
+FROM one_hour_blocks b
+INNER JOIN coinbase c ON c.block_id = b.block_id
+GROUP BY bucket
+ORDER BY 1;
+```
+![fees and block reward](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/revenue_ratio.png)
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:314]:
```suggestion
![Line graph with two lines showing the average fee and block reward, over the last five days](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/revenue_ratio.png)
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 13]
```diff
@@ -0,0 +1,393 @@
...
+This chart analyzes the last 5 days of average miner revenue. The left
+axis is a percentage value that indicates the portion of total revenue that
+comes from transaction fees (green) and block rewards (yellow). It might be
+surprising for some, but most miner revenue comes from block rewards
+(6.25BTC at the moment). The fees portion never went above 3 % in the
+last 5 days. 
+
+This kind of analysis can start discussions around the long-term fading of
+block rewards and how on-chain fees will need to go up in the future to
+incentivize the miners and sustain the network. (Note that the left axis is
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:326]:
```suggestion
block rewards and how on-chain fees need to rise to
incentivize miners and sustain the network. (Note that the left axis is
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 14]
```diff
@@ -0,0 +1,393 @@
...
+)
+SELECT
+   bucket as "time",
+   average_y(rolling(block_stats) OVER (ORDER BY bucket RANGE '12 hours' PRECEDING)) AS "block weight",
+   average_x(rolling(block_stats) OVER (ORDER BY bucket RANGE '12 hours' PRECEDING))*0.00000001 AS "mining fee"
+FROM stats
+ORDER BY 1
+```
+
+![block weight and fees](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/weight_fees.png)
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:355]:
```suggestion
![block weight and fees](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/weight_fees.png)
```
Am I misinterpreting something, or is this not the graph produced from the query? Graph is titled 4-week moving average, and x-axis spans less than a day

Also this section seems very similar to the weight vs. fee section above, but using a rolling average. Is there a reason for having both, or should they be consolidated into one?

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 15]
```diff
@@ -0,0 +1,393 @@
...
+
+You can see that the block weight and block mining fee are indeed tightly
+connected to each other. In practice, you can also see that four million
+weight units are the size limit introduced in the 2017’s SegWit update.
+This means that, looking at this graph, there’s still room to grow for
+individual blocks, and they could include even more transactions.
+
+### What’s the average miner revenue per block?
+Now let’s analyze how much revenue miners actually generate by mining a
+new block on the blockchain, including fees and block rewards. This query
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:365]:
```suggestion
Now, analyze how much revenue miners actually generate by mining a
new block on the blockchain, including fees and block rewards. This query
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 16]
```diff
@@ -0,0 +1,393 @@
...
+
+```sql
+SELECT
+   bucket as "time",
+   average_y(rolling(stats_miner_revenue) OVER (ORDER BY bucket RANGE '12 hours' PRECEDING))*0.00000001 AS "revenue in BTC"
+FROM one_hour_coinbase
+WHERE bucket > NOW() - INTERVAL '1 day'
+ORDER BY 1
+```
+![miner revenue per block](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/miner_revenue_per_block.png)
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:376]:
```suggestion
![Average miner revenue per block, plotted over the last day](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/miner_revenue_per_block.png)
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 17]
```diff
@@ -0,0 +1,393 @@
...
+(THIS CHART NEEDS AN UPDATE - larger time frame?)
+
+To add some spice to the chart, let’s add the BTC-USD rate to the analysis
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:379]:
```suggestion
To make the chart more interesting, add the BTC-USD rate to the analysis
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 18]
```diff
@@ -0,0 +1,393 @@
...
+SELECT
+   bucket as "time",
+   average_y(rolling(stats_miner_revenue) OVER (ORDER BY bucket RANGE '7 days' PRECEDING))*0.00000001 AS "revenue in BTC",
+   average_x(rolling(stats_miner_revenue) OVER (ORDER BY bucket RANGE '7 days' PRECEDING)) AS "revenue in USD"
+FROM coinbase_one_day
+WHERE
+ bucket > NOW() - INTERVAL '5 day'
+ORDER BY 1
+```
+![miner revenue per block](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/miner_revenue_per_block_with_btcusd.png)
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:392]:
```suggestion
![Average miner revenue per block, plotted in BTC and USD, over the last five days](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/miner_revenue_per_block_with_btcusd.png)
```

**Review by charislam — COMMENTED** [review]
Great writing on this! Very clear.

Some recurrent points that I changed:
- Many instances of "let's". We try to avoid first person in the docs, unless we are specifically speaking _as Timescale_. Using first person in instructions results in a weird mix of 'we' and 'you' doing these actions, so I removed these.
- The text that goes in Markdown image embeds shows up as alt text in the final HTML. I expanded these to make them more descriptive and accessible.

File: timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md [hunk 19]
```diff
@@ -0,0 +1,179 @@
...
+-------------------|----------------------------------------------------------------|--------|------|
+2022-05-30 01:42:17|6543a8e489eade391f099df7066f17783ea2f9d19d644d818ac22bd8fb86005e|  738489|   863|
+2022-05-30 01:42:17|a9e2bb3e734e0c0535da4e8ab6e3d0352a44db443d48a861bd5b196575dfd3ff|  738489|   577|
+2022-05-30 01:42:17|fd0a9a8c31962107d0a5a0c4ef2a5702e2c9fad6d989e7ac543d87783205a980|  738489|   758|
+2022-05-30 01:42:17|e2aedc6026459381485cd57f3e66ea88121e5094c03fa4634193417069058609|  738489|   766|
+2022-05-30 01:42:17|429c0d00282645b54bd3c0082800a85d1c952d1764c54dc2a591f97b97c93fbd|  738489|   766|
+
+<highlight type="note">
+A **coinbase transaction** is the first transaction in each block in which the
+miner receives the transaction fees and block reward for the mining work.
```
[the remark this answers is not in the document]
Comment (zseta) [reply timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md:140]: thanks I edited your suggestion. Coinbase transaction is technically the first transaction, but the important bit here is that this transaction goes to the miners (other transactions don't they are just regular BTC transactions.)

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 20]
```diff
@@ -0,0 +1,385 @@
...
+either part of the TimescaleDB extension or the Toolkit extension. You need to
+install and enable the Toolkit extension to successfully run the following
+queries.
+
+Enable Toolkit extension:
+```sql
+CREATE EXTENSION timescaledb_toolkit;
+```
+
+YOu have everything now to create the continuous aggregates and start analyzing!
```
Comment (ryanbooz) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:38]:
```suggestion
You have everything now to create the continuous aggregates and start analyzing!
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 21]
```diff
@@ -0,0 +1,385 @@
...
+you get a relevant SQL query and a chart that answers the question.
+
+**Questions**
+* Is there any connection between the number of transactions and the transaction fees?
+* Does the transaction volume affect the BTC-USD rate?
+* Do more transactions in a block mean the block is more expensive to mine?
+* What percentage of the average miner's revenue comes from fees vs. block rewards?
+* How does block weight affect miner fees?
+* What’s the average miner revenue per block?
+
```
Comment (ryanbooz) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:189]: This feels unnecessary to me since we don't link to anything. Any other opinions?

Comment (zseta) [reply timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:189]: I can add anchor links so the reader can jump to the one they find the most interesting

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md (hunk 14 above)
Comment (zseta) [reply timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:344]: correct good catch let me create the proper chart

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 22]
```diff
@@ -0,0 +1,393 @@
...
+   bucket as "time",
+   avg(block_fee_sat)*0.00000001 AS "fees",
+   FIRST((c.coinbase_tx - block_fee_sat), bucket)*0.00000001 AS "reward"
+FROM one_hour_blocks b
+INNER JOIN coinbase c ON c.block_id = b.block_id
+GROUP BY bucket
+ORDER BY 1;
+```
+![fees and block reward](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/revenue_ratio.png)
+(THIS CHART NEEDS AN UPDATE)
```
Comment (zseta) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:304]:
```suggestion
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 23]
```diff
@@ -0,0 +1,393 @@
...
+install and enable the Toolkit extension to successfully run the following
+queries.
+
+Enable Toolkit extension:
+```sql
+CREATE EXTENSION timescaledb_toolkit;
+```
+
+YOu have everything now to create the continuous aggregates and start analyzing!
+
```
[the remark this answers is not in the document]
Comment (zseta) [reply timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md]: agree!

[the remark this answers is not in the document]
Comment (zseta) [reply timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md]: https://github.com/timescale/docs/pull/1176/commits/4faf19c6e7ac5f928fec8d87bfe4371b127f23f8

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md (hunk 21 above)
Comment (zseta) [reply timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md]: https://github.com/timescale/docs/pull/1176/commits/250b638967d8539d5237a70b48dd1a66c186ff20

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 24]
```diff
@@ -0,0 +1,389 @@
...
+   average_x(rolling(block_stats) OVER (ORDER BY bucket RANGE '12 hours' PRECEDING))*0.00000001 AS "mining fee"
+FROM stats
+ORDER BY 1
+```
+
+![block weight and fees](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/weight_fees.png)
+
+You can see that the block weight and block mining fee are indeed tightly
+connected to each other. In practice, you can also see that four million
+weight units are the size limit introduced in the 2017’s SegWit update.
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:351]:
```suggestion
weight units is the size limit introduced in the 2017 SegWit update.
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md [hunk 25]
```diff
@@ -0,0 +1,389 @@
...
+```sql
+SELECT
+   bucket as "time",
+   average_y(rolling(stats_miner_revenue) OVER (ORDER BY bucket RANGE '12 hours' PRECEDING))*0.00000001 AS "revenue in BTC"
+FROM one_hour_coinbase
+WHERE bucket > NOW() - INTERVAL '1 day'
+ORDER BY 1
+```
+![Average miner revenue per block, plotted over the last day](https://assets.timescale.com/docs/images/tutorials/bitcoin-blockchain/miner_revenue_per_block.png)
+(THIS CHART NEEDS AN UPDATE - larger time frame?)
```
Comment (charislam) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md:369]: don't know if you're ready to merge but if you are just a friendly reminder that this is here :)

**Review by charislam — APPROVED** [review]
Looks great!

File: timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md (hunk 25 above)
Comment (zseta) [reply timescaledb/tutorials/analyze-bitcoin-blockchain/analyze-blockchain.md]: Thx I missed that, the chart is fine just needed to fix the SQL code

**Review by ryanbooz — APPROVED** [review]

File: timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md [hunk 26]
```diff
@@ -0,0 +1,176 @@
...
+You created the hypertable and added proper indexes.
+Next, ingest some Bitcoin transactions. The sample data file
+contains Bitcoin transactions from the past five days. This CSV file is
+updated daily so you always download recent Bitcoin transactions.
+Insert this dataset into your TimescaleDB instance.
+
+<procedure>
+
+### Ingesting Bitcoin transactions
+1. Download the sample `.csv` file: <tag type="download">[bitcoin_sample.csv](https://assets.timescale.com/docs/downloads/bitcoin-blockchain/bitcoin_sample.zip)</tag>
```
Comment (zseta) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md:96]:
```suggestion
1. Download the sample `.csv` file: <tag type="download">[tutorial_bitcoin_sample.csv](https://assets.timescale.com/docs/downloads/bitcoin-blockchain/bitcoin_sample.zip)</tag>
```

File: timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md [hunk 27]
```diff
@@ -0,0 +1,176 @@
...
+1. Unzip the file and change the directory if you need to:
+    ```bash
+    unzip bitcoin_sample.zip
+    cd bitcoin_sample
+    ```
+1. At the `psql` prompt, insert the content of the `.csv` file into the database.
+    ```bash
+    psql -x "postgres://tsdbadmin:<YOUR_PASSWORD_HERE>@<YOUR_HOSTNAME_HERE>:<YOUR_PORT_HERE>/tsdb?sslmode=require"
+    
+    \COPY transactions FROM 'bitcoin_sample.csv' CSV HEADER;
```
Comment (zseta) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md:109]:
```suggestion
    \COPY transactions FROM 'tutorial_bitcoin_sample.csv' CSV HEADER;
```
